src/static/helpers/index.ts

```diff
@@ -8,6 +8,7 @@
 // Doing so may lead to circular dependencies or duplicate exports (due to rollup mangling the types)
 export * from './environment';
 export * from './slasHelper';
+export * from './passwordlessHelper';
```

Comment:
NIT: The passwordless login API's are part of SLAS right? And from the looks of it, the content of this file is somewhat small, is there a reason you didn't slap it in the slasHelpers, sounds like that is the right context for it.

Reply:
Thats super lame, seems a little arbitrary. 

It definitely seems like the SLAS passwordless helper should be located along with all the other SLAS helpers. Can you propose an alternative (non-breaking) solution here?

Reply:
Agreed that it's pretty arbitrary, when the line limit was implemented (3 years ago) I don't think the team originally envisioned a helper file growing to this size. I think it'd be ok to update the line limit or ignore it for this file:

Ignore: `/* eslint-disable max-lines */`
Update max lines: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/2af2671a69b6e7b656d91cbc5b319cd1066eeb33/.eslintrc.json#L57

Reply:
I'll remove the maxlines to make it as flexible as possible for new helpers as part of this epic